development/webpack/test/plugins.ManifestPlugin.test.ts

```diff
@@ -281,4 +281,116 @@ describe('ManifestPlugin', () => {
       }
     }
   });
+
+  describe('manifest flags in development mode', () => {
```

Comment:
You still need to restore the originals after each test via `afterEach(() => mock.restoreAll());`

previous discussion: https://github.com/MetaMask/metamask-extension/pull/29696/files/b09f6a5687d494fa4029d067fd9b831ab1ee4a19#r1954768430

Reply:
Aahhh i missed this one, restored here d8fb679ab8d4a6500bc1b45f91b29b88f6400dad